Tests/PackageLoading/ConventionTests.swift

```diff
@@ -440,6 +440,95 @@ class ConventionTests: XCTestCase {
         }
     }
 
+    func testValidSources() throws {
+        var fs = InMemoryFileSystem()
+        try fs.createEmptyFiles("/main.swift",
```

Comment:
I think we should, I like how you can quickly create a package. However we do have a proper init now and this becomes untidy once your package grows.
maybe @czechboy0 has some stats about how much this convention is used?